### ISSUE TITLE
Screen reader is announcing “Grouping” when navigating through chat history in browse mode.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5213]
### Is it an issue related to Adaptive Cards?

No.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

1. Open [https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true](https://wwwppe.microsoft.com/en-us/surface) in Edge. 
2. Start NVDA. 
3. [Tab] to the ‘Help me choose’ button and open the window. Focus will be on the close “X”. 
4. Tab x 2 
5. “Chat history, use the up and down arrow keys” 
6. “Grouping Bot Surface said” – Is it clear to the user what this means? 

### What do you expect?

Screen readers should not announce grouping information while navigating through chat history.
Remove the role=”group” from the <div>.

### What actually happened?

Screen reader is announcing “Grouping” when navigating through chat history in browse mode.

### Do you have any screenshots or recordings to repro the issue?

<img width="959" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/1f4b65b1-441b-4d0f-9f3a-2923f4715fe8">

### MAS reference

https://aka.ms/MAS1.3.1

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#AILimited;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;
